app/views/support/WeeklyPromotion.scala

```diff
@@ -168,45 +181,46 @@ object WeeklyPromotion {
 
 }
 
-object PlanPicker {
+object WeeklyPicker {
 
   import model.GuardianWeeklyZones._
 
-  def showUpdatedPrices = {
+  val updateTime = DateTime.parse("2018-10-10T09:45:00").withZone(DateTimeZone.UTC)
+
+  def showUpdatedPrices(forceShowUpdatedPrices: Boolean, timeToUpdate: DateTime = updateTime): Boolean = {
```

Comment:
All of the logic for determining whether to display the new prices now happens in showUpdatedPrices, which makes it easier to test each function in isolation.